rafthttp/transport.go

```diff
@@ -307,3 +329,23 @@ func (t *Transport) Resume() {
 		p.(Pausable).Resume()
 	}
 }
+
+// snapshotStore is the temporary store of snapshot. Caller could put one
```

Comment:
i see your point. the store is not a temporary one and it exists in the whole life cycle of Transport. Calling it `store of temporary snapshot` is weird too because the snapshot has its own life cycle. I guess I will just delete it.